src/models/room.ts

```diff
@@ -2939,6 +2939,21 @@ export class Room extends TypedEventEmitter<EmittedEvents, RoomEventHandlerMap>
         }
         event.applyVisibilityEvent(visibilityChange);
     }
+
+    /**
+     * Determines whether the given room is a sip room.
+     * @returns {boolean} True if the given the given room is a sip room.
+     */
+    public isSip(): boolean {
+        for (const member of this.getMembers()) {
+            if(member.userId != this.client.getUserId()){
+                if(member.userId.startsWith("@_sip_")){
```

Comment:
I don't think it's specced, how would you purpose we go about this?
Our users are all created with this prefix as per the config on the sip bridge, since it's not possible to change a mxid I think it's reasonable to hard code this for now. 
https://gitlab.matrix.org/new-vector/voip/matrix-sip-bridge/-/blob/master/config_sample.yaml#L27

Do you perhaps have a better succession? 🙈

Reply:
That's specific to your config on that sip bridge, other sip bridges may have different config. SIP is also not a concept mentioned in the Matrix spec at all. This really cannot land in its current state, it'd be better done as part of the extensible profiles project https://github.com/matrix-org/matrix-spec-proposals/issues/3795

Reply:
If you want room types you don't need a spec change, you can use a room type like `foo.chagai95.sip`

Reply:
Ok cool, would you merge such a room type?

Reply:
You could always just do `Room::getType() === `whatever.sip` - it doesn't need to be `Room::isSip()` based on a non-public SIP bridge

Reply:
Ok thx, would you merge something like that into the react SDK?

Reply:
If it was just a bit of dead code probably not, if it was something more generic to hide rooms with types the app doesnt know then probably

Reply:
Ok thanks! It's about creating a call list out of the sip rooms.

Reply:
@t3chguy I hope it's ok for me to ping you, I was just wondering why not use m.bridge? I've seen that used in Beeper...

Reply:
![image](https://user-images.githubusercontent.com/2403652/172110553-e2e2d0b2-d4cd-4502-a15e-2791e0d362d6.png)

Just because one vendor uses it, doesn't make it specced. This is especially bad given the `m.` prefix which is reserved for specced things only

Reply:
Ok, thx for clarifying that!